app/src/main/java/com/sunlightlabs/android/congress/fragments/LegislatorListFragment.java

```diff
@@ -407,7 +407,7 @@ public LoadLegislatorsTask(LegislatorListFragment context) {
 					temp = CommitteeService.find(context.committee.id).members;
 					break;
 				case SEARCH_STATE:
-					temp = LegislatorService.allWhere("state", context.state);
+					temp = LegislatorService.allForState(context.state);
```

Comment:
The `allWhere` function, which is tied to the Sunlight API, is still used for searching by last name (which I think we'll have to do by filtering client side?) and when searching by chamber (which will be implemented on a future update).